lib/operations/logic/equal.js

```diff
@@ -0,0 +1,7 @@
+function equal(a, b) {
+  // eslint-disable-next-line eqeqeq
+  return a == b;
+}
+
+equal.code = '==';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`==` is an existing operator